src/bitcoin-cli.cpp

```diff
@@ -67,10 +67,10 @@ static void SetupCliArgs(ArgsManager& argsman)
 {
     SetupHelpOptions(argsman);
 
-    const auto defaultBaseParams = CreateBaseChainParams(CBaseChainParams::MAIN);
-    const auto testnetBaseParams = CreateBaseChainParams(CBaseChainParams::TESTNET);
-    const auto signetBaseParams = CreateBaseChainParams(CBaseChainParams::SIGNET);
-    const auto regtestBaseParams = CreateBaseChainParams(CBaseChainParams::REGTEST);
+    const auto defaultBaseParams = kernel::CreateBaseChainParams(CBaseChainParams::MAIN);
```

Comment:
nit: Could use `using` declarations in `.cpp` files (if you want)

Reply:
Seems to me like `using` declarations are not used for internal namespaces in the code so far, so I chose not to.